src/react/components/PopupEditTable.js

```diff
@@ -48,21 +51,21 @@ const DisplaySetHandler = ({ sets, setInputChanged }) => {
   );
 };
 const DisplaySet = ({ set, index, setInputChanged }) => {
-  let [player1Set, setPlayer1Set] = useState(set.player1);
-  let [player2Set, setPlayer2Set] = useState(set.player2);
+  const [player1Set, setPlayer1Set] = useState(set.player1);
+  const [player2Set, setPlayer2Set] = useState(set.player2);
   let [css, setCss] = useState("");
 
   const checkInput1 = event => {
     setPlayer1Set(event.target.value);
 
-    console.log(player1Set + " " + player2Set);
     if (
-      (player1Set === 11 && player2Set < 10) ||
-      (player2Set === 11 && player1Set < 10) ||
-      (player1Set >= 10 && player1Set - player2Set === 2) ||
-      (player2Set >= 10 && player2Set - player1Set === 2)
+      (event.target.value === 11 && player2Set < 10) ||
+      (player2Set === 11 && event.target.value < 10) ||
+      (event.target.value >= 9 && player2Set - event.target.value === 2) ||
+      (player2Set >= 9 && event.target.value - player2Set === 2)
```

Comment:
kannst du das vielleicht in eine neue Funktion auslagern damit klarer ist, was du hier prüfst?

Reply:
done :)